### PR TITLE
gameboy.xml, nes.xml, snes.xml

### PR DESCRIPTION
**Pull request by quasiscroto**
fix miscellaneous clones, non-clones, typos, and a software name error

### CODE REVIEW

File: hash/snes.xml [hunk 1]
```diff
@@ -44636,7 +44636,7 @@ List of unclassified roms
 		</part>
```
Comment (MoochMcGee) [inline hash/snes.xml]: Please remove this line, it's a violation of the XML standard.

Comment (MooglyGuy) [reply hash/snes.xml]: @MoochMcGee Please stop reviewing other peoples' pull requests. You are not a MAME developer, you are an external contributor. Reviewing pull requests is the purview of the core team only.

Comment (angelosa) [reply hash/snes.xml]: If that's really the case then I do believe that you can disable commentary by role in github settings (is it?)

Comment (happppp) [reply hash/snes.xml]: No, anyone should be allowed to comment. External contributors need to be careful with their wording to not come across as they have the right to make demands. Especially @MoochMcGee who has been banned before for making rude comments on a PR.

On-topic: I see nothing wrong with snes.xml line 44636

Comment (Tafoid) [reply hash/snes.xml]: I've scanned the new files with xmllint - no format errors are showing up in that or in my check for TAB/Tag formatting.

Comment (cuavas) [reply hash/snes.xml]: The problem is that GitHub is displaying it strangely if you expand the diff in the browser – it makes it look like there’s a stray `</part>` when there isn’t.  GitHub still gets weird when viewing large files.

**Comment by smf-** [comment]
On 30/01/2020 14:03, Vas Crabb wrote:
>
> *@cuavas* commented on this pull request.
>
> ------------------------------------------------------------------------
>
> In hash/snes.xml
> <https://github.com/mamedev/mame/pull/6220#discussion_r372965139>:
>
> > @@ -44636,7 +44636,7 @@ List of unclassified roms
>
>   		</part>
>
> The problem is that GitHub is displaying it strangely if you expand
> the diff in the browser – it makes it look like there’s a stray
> |</part>| when there isn’t. GitHub still gets weird when viewing large
> files.
>
I don't see the problem. There is a unified diff hunk identifier above
it, it's clearly the closing tag of the previous xml node before the change.

**Comment by cuavas** [comment]
If you hit the thing to expand it, GitHub gets confused.  If you look at the line numbers you can see the discontinuity, but there's no visual divider (see attached).
clipping

**Pull request merged**